test/helpers/foundry/helpers.ts

```diff
@@ -355,46 +389,56 @@ async function extractFromZip(
     },
   };
 
-  const { files } = await Unzip.custom(source);
-  const filtered = files.filter(({ path }) =>
+  const directory = await (Unzip as unknown as UnzipStatic).url(source, {});
```

Comment:
In my PR `Unzip` has custom types defined so we don't need casting: https://github.com/MetaMask/metamask-extension/blob/foundryup/types/unzipper.d.ts

Reply:
mmm so I added a custom types because lint was failing when removing "any" for the source type :/ I can review that again